src/fireflyClient.ts

```diff
@@ -713,6 +713,7 @@ export class FireflyClient {
         symbol: params.symbol,
         orderHashes: params.hashes,
         cancelSignature: params.signature,
+        parentAddress: params.parentAddress
```

Comment:
what if I am the parent myself? in that case parent address will be undefined?

Reply:
As discussed , undefined is fine .